db/migrate/20181130072917_remove_foreign_key_events_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveForeignKeyEventsAccounts < ActiveRecord::Migration
+  def change
+    remove_foreign_key :event_store_events, column: :provider_id
```

Comment:
We should at least have an index on this column

Reply:
Ok 😉 An index would do 👍

Reply:
I see the `index` already exists and I am not removing it:
https://github.com/3scale/porta/blob/8150fda5b5a03ddc8d37ad875c81955c6173fd2d/db/schema.rb#L486